diff -urN libview-0.6.4/libview.pc.in libview-0.6.4.new/libview.pc.in
--- libview-0.6.4/libview.pc.in	2008-11-26 01:31:17.000000000 +0100
+++ libview-0.6.4.new/libview.pc.in	2009-10-21 22:32:06.357221460 +0200
@@ -1,9 +1,11 @@
-prefix=${prefix}
-libdir=${exec_prefix}/lib
-includedir=${prefix}/include
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
 
 Name: libview
 Description: VMware's Incredibly Exciting Widgets for GTK+
+Requires: gtkmm-2.4 sigc++-2.0
 Version: @VERSION@
-Libs: -L${libdir} -lview @PACKAGE_LIBS@
-Cflags: -I${includedir} @PACKAGE_CFLAGS@
+Libs: -L${libdir} -lview
+Cflags: -I${includedir}
